lib/widgets/molecules/invitation_section.dart

```diff
@@ -21,43 +23,65 @@ class InvitationSection extends StatefulWidget {
 }
 
 class _InvitationSectionState extends State<InvitationSection> {
+  ChannelsProvider? _channelsProvider;
+  AppLocalizations? l10n;
+  Future<OperationResult<InvitationInbox>>? _invitationInbox;
+
+  @override
+  void initState() {
+    super.initState();
+    _channelsProvider = Provider.of<ChannelsProvider>(context, listen: false);
+  }
+
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    _invitationInbox = _channelsProvider?.getInboxInvitations();
+    l10n = AppLocalizations.of(context);
```

Comment:
Is the context available here?

Reply:
Yes, the BuildContext is an instance field of the State class. It is available right after initialization.